tests/Wrapper/ProductTest.php

```diff
@@ -847,6 +847,85 @@ public function testCallistoUrlFormatIsUsedWhenConfigured(): void
         );
     }
 
+    public function itemShowPleaseSelectProvider()
```

Comment:
Add missing typehint

```suggestion
    public function itemShowPleaseSelectProvider(): array
```